queue_services/entity-filer/src/entity_filer/filing_processors/amalgamation_application.py

```diff
@@ -18,11 +18,30 @@
 from typing import Dict
 
 import sentry_sdk
-from business_model import AmalgamatingBusiness, Amalgamation, Document, Filing, LegalEntity, RegistrationBootstrap, db
+from entity_queue_common.service_utils import QueueException
+from legal_api import db
+from legal_api.models import (
```

Comment:
The filer imports it's models from the `business_model` library, not from legal_api. This can also be updated for the filer in `tests/unit/__init__.py` and `test_amalgamation_application.py`. 

Also, you can probably remove the `AccountService` import from line 32 in this file since it isn't used.